data.sql

```diff
@@ -5,3 +5,15 @@ INSERT into animal (name, date_of_birth, escape_attempts, neutered, weight_kg)VA
 INSERT into animal (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', '2018-11-15', 2, true, 8);
 
 INSERT into animal (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', '2017-05-12', 5, true, 11);
+
+
+-- Day 2 project
+select * from animals;
+alter table animals add column species varchar;
+INSERT INTO animal (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Charmander', '2020/2/8', '0', 'false', '-11.0');
+INSERT into animal (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Plantmon', '2021-11-15', 2, true, -5.7);
+INSERT into animal (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Squirtle', '1993-04-02', 3, false, -12.13);
+INSERT into animal (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Angemon', '2005-06-12', 1, true, -45);
+INSERT into animal (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Boarmon', '2005-06-07', 7, true, 24.5);
```

Comment:
- Seems like you added the wrong data here, according to the project requirements, the weight is 20.4. Please fix this 🙏